src/commands/config/levelRole/index.ts

```diff
@@ -46,20 +50,32 @@ const command: Command = {
         },
       }
 
-    await Config.configLevelRole(msg, {
+    const res = await Config.configLevelRole(msg, {
       guild_id: msg.guildId,
       role_id: role.id,
       level,
     })
-
-    return {
-      messageOptions: {
-        embeds: [
-          composeEmbedMessage(msg, {
-            description: `Successfully configured role <@&${role.id}> for level ${level}`,
-          }),
-        ],
-      },
+    if (res.ok) {
+      return {
+        messageOptions: {
+          embeds: [
+            getSuccessEmbed({
+              msg,
+              description: `Successfully configured role <@&${role.id}> for level ${level}`,
+            }),
+          ],
+        },
+      }
+    } else {
+      let errEmbed = getErrorEmbed({ msg })
+      if (res.error.includes("Role has been used")) {
+        errEmbed = getErrorEmbed({ msg, description: res.error })
+      }
+      return {
+        messageOptions: {
+          embeds: [errEmbed],
+        },
+      }
```

Comment:
```suggestion
      let description
      if (res.error.toLowerCase().includes("Role has been used")) {
        description = res.error
      }
      return {
        messageOptions: {
          embeds: [getErrorEmbed({ msg, description })],
        },
      }
```
`description` sẽ là `string` (nếu error role has been used... gì đó) hoặc undefined (và nó sẽ fallback về generic message)